docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -102,3 +102,32 @@ Each round represents one iteration of the training loop in the federated settin
 * The weight updates are aggregated by the aggregator organization.
 * The aggregated organization send the aggregated updates to the training organizations.
 * The training organizations update their model with the aggregated updates.
+
+Centralized strategy - workflow
+--------------------------------
+
+The workflow of a centralised strategy, unless specified otherwise, is as follows:
+
+- initialisation round: one train task on each train organization
+- then for each round: one aggregate task on the central organization then one train task on each train organization
+
+Steps of an aggregate task:
+
+- Calculate the common shared state from the previous train tasks shared state.
```

Comment:
the common shared state is always the average of the gradients + some parameters for strategies like SCAFFOLD?

Reply:
not exactly, may vary

Reply:
Ok. Then do you think we can give examples of what is in shared state, for instance of the FedAvg strategy?

Reply:
Is the general workflow and message clear without going into examples?
I would rather keep things general here and when we have an actual strategy description somewhere, add a link to that page